src/lib/es5.d.ts

```diff
@@ -1,8 +1,8 @@
+/// <reference lib="es3" />
 /////////////////////////////
 /// ECMAScript APIs
 /////////////////////////////
 
-declare const NaN: number;
 declare const Infinity: number;

```

Comment:
@saschanaz Shouldn't the Infinity constant also be shifted into `es3.d.ts`? See section 4.3.22 of [the spec](https://www.ecma-international.org/publications/files/ECMA-ST-ARCH/ECMA-262,%203rd%20edition,%20December%201999.pdf).